<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>event1</class>
 <widget class="QDialog" name="event1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1647</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 75 10pt &quot;MV Boli&quot;;
}
#event1
{

background-color: QLinearGradient(x1: 0,y1: 0,x2: 0,y2: 1,stop: 0 #e66465, stop:1 #9198e5);
border-radius:15px;
}
QLabel
{
	color: rgb(0, 0, 0);
}
QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
QPushButton
{
border-radius:15px;
}
QPushButton:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}
Qlabel{
height:50px;
width:40px;
}

QToolButton
{
background: transparent;
border-radius:60px;

}</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>1121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0)
</string>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Localisation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Calendrier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 0);

font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion des Activités</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1981</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="QLabel" name="Numero">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Numero">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Nom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date_evenement">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date_Activités</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lieu">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lieu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Lieu">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nombredesparticipants">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre_des_participants</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nombre_des_participants">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="Actions">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Actions</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Actions">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouter">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modifier">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QTableView" name="tab_evenement">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>1151</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>580</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_tri_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>580</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tri</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_chercher">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Chechez un Activités par Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>570</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>     Supprimer un Activités</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_supprimer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>610</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stat">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>580</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stat</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Déconnecter</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../logout.png</normaloff>../logout.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
